R/lru.R

```diff
@@ -52,25 +63,21 @@ LRUcache_ <- R6::R6Class("LRUcache_",
       }
       # Check for eviction
       while (private$check_for_eviction(name, size)) { private$evict() }
-      private$data[[name]] <- list(value = value, timestamp = Sys.time())
+      private$data[[name]] <- list(value = value, last_get = time, last_set = time, size = size)
```

Comment:
I personally don't think the value of renaming `timestamp` is worth the breaking change.

Reply:
Agreed. Will refactor.